assets/out

```diff
@@ -186,6 +188,26 @@ elif [ -n "$build" ]; then
     fi
   fi
 
+  label_keys=($(echo "$labels" | jq -r 'keys | join(" ")'))
+  if [ "${#label_keys[@]}" -gt 0 ]; then
+    for key in "${label_keys[@]}"; do
+      value=$(echo "$labels" | jq -r --arg "k" "$key" '.[$k]')
+      expanded_build_args+=("--label")
+      expanded_build_args+=("${key}=${value}")
+    done
+  fi
+
+  if [ -n "$labels_file" ]; then
+    labels_keys=($(jq -r 'keys | join(" ")' "$labels_file"))
+    if [ "${#labels_keys[@]}" -gt 0 ]; then
+      for key in "${labels_keys[@]}"; do
+        value=$(jq -r --arg "k" "$key" '.[$k]' "$labels_file")
+        expanded_build_args+=("--label")
+        expanded_build_args+=("${key}=${value}")
```

Comment:
I reused `expanded_build_arg` to pass the variables, should the labels have their own variable? e.g. `expanded_labels`

Reply:
Maybe we could just give it a more general name? Looks like the `target` var could also just use the same var.